policies/modules/ROOT/nav.adoc

```diff
@@ -30,7 +30,7 @@
 *** xref:policies::policies-included-pingfederate-oauth-token-enforcement.adoc[PingFederate OAuth 2.0 Token Enforcement]
 *** xref:policies::policies-included-rate-limiting.adoc[Rate Limiting]
 *** xref:policies::policies-included-rate-limiting-sla.adoc[Rate Limiting: SLA-Based]
-*** xref:policies::policies-included-schema-validation.adoc[Schema Validation (OAS3)]
+*** xref:policies::policies-included-schema-validation.adoc[OAS3 Schema Validation Policy]
```

Comment:
```suggestion
*** xref:policies::policies-included-schema-validation.adoc[OAS3 Schema Validation]
```

Also, since the title now starts with "O", please place this line in the appropriate place in the overall list (it's all alphabetical)